pkg/skaffold/kubernetes/portforward/entry_manager_test.go

```diff
@@ -101,6 +101,14 @@ func TestForwardedPorts(t *testing.T) {
 		t.Fatal("didn't load port 9000 correctly")
 	}
 
+	if _, ok := pf.LoadOrStore(4000, struct{}{}); ok {
```

Comment:
I know this isn't directly related to this change, but it got me thinking. do we really need the signature of this method to take a `struct{}{}` at all? aren't we just using this as an atomic set to track which ports are in use? maybe we could just change the signature to `LoadOrStore(value int)` and have the implementation throw the `struct{}{}` in there. WDYT?

Reply:
So this is necessary because the `forwardedPorts` type has to implement the [ForwardedPorts](https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/util/port.go#L30) interface so that it can use [GetAvailablePort](https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/util/port.go#L410).

I think the event API uses that function as well, except with a `sync.Map`, so we need `LoadOrStore` as written to work for both!

Reply:
:+1: thanks for clarifying!